File: AlarmManager/app/src/main/java/com/example/bob/alarmmanager/MainActivity.java
package com.example.bob.alarmmanager;

import android.app.AlarmManager;
import android.app.PendingIntent;
import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
/*
    Copy pasted from:
    https://androidclarified.com/android-example-alarm-manager-complete-working/
*/
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final int REQUEST_CODE=101;

        Intent intent = new Intent(this, MyReceiver.class);

        PendingIntent pendingIntent = PendingIntent.getBroadcast(
                this,
                        REQUEST_CODE, intent,
                        PendingIntent.FLAG_UPDATE_CURRENT);

        AlarmManager alarmManager = (AlarmManager) getSystemService(ALARM_SERVICE);
        alarmManager.set(AlarmManager.ELAPSED_REALTIME, 10000, pendingIntent);
    }
}

File: CodeKataForFragments/app/src/main/java/com/example/bob/codekataforfragments/MainActivity.kt
package com.example.bob.codekataforfragments

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var fragOne = FragOne()
        var fragTwo = FragTwo()

        supportFragmentManager.beginTransaction()
                .add(R.id.fragment_container_one, fragOne)
                .add(R.id.fragment_container_two, fragTwo)
                .commit()
    }
}
